x-pack/solutions/search/plugins/search_indices/public/components/shared/create_index_panel.tsx

```diff
@@ -33,7 +33,6 @@ export interface CreateIndexPanelProps {
   createIndexView: CreateIndexViewMode;
   onChangeView: (id: string) => void;
   onClose: () => void;
-  showCallouts?: boolean;
```

Comment:
We should not remove this, this was used to hide the callouts when we use this panel on the create index page. Where they do not make as much sense as on the start page. Did we decide we always want to show them on both the start and create index pages?

Reply:
hey, yes. we decided to show them when there are indices and where there are no indices.

Reply:
For both hosted and serverless?